mods/cnc/sequences/decorations.yaml

```diff
@@ -194,6 +231,7 @@ t15:
 			WINTER: t15.win
 			SNOW: t15.sno
 	idle:
+		Palette: staticterrain
```

Comment:
Can we move these to the `Defaults` node?

Reply:
I assume manually you'd move many of these to `Default` nodes